### DISCUSSION TITLE
Oqtane Presentations

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#3250] [General]
I did a general Oqtane presentation On .NET Live today:

https://www.youtube.com/watch?v=A7KnkHe3AXY

Tomorrow at 1:15 PM ET (10:15 AM PT) during the .NET Foundation Summit I will be doing a presentation focused on the Oqtane Marketplace:

https://lnkd.in/eGiuHUf9

**Comment by leigh-pointer** [comment]
And it was super!

**Comment by thabaum** [comment]
![image](https://github.com/oqtane/oqtane.framework/assets/13577556/8f849427-7835-4f65-89a9-8160aba3f917)

Blast from the past ;) I remember buying this back in the day!

I am hoping to build a migration script to Oqtane form DNN, Wordpress, Drupal and others at some point to help others migrate simple sites over.  I will probably make that an opensource project unless someone can please beat me to it.

**Reply by sbwalker** [reply]
Mitchel Sellers and I are now working together again on the .NET Foundation :)

**Reply by thabaum** [reply]
Oqtane ReUnited :)

**Comment by oqtanecms** [comment]
Oqtane will be become more popular and powerful than dotnetnuke ！

**Reply by sbwalker** [reply]
:) I believe Oqtane already is more powerful than DNN as it is multi-platform, multi-device, multi-database, and supports both isolated and shared tenancy models

**Reply by leigh-pointer** [reply]
I loved DNN in the time but Oqtane is lightyears further advanced than DNN.
